.github/workflows/render-readme.yaml

```diff
@@ -43,7 +43,6 @@ jobs:
             any::cranlogs
             any::lubridate
             any::ggplot2
-            any::poorman
```

Comment:
I think it shouldn't be removed from this workflow, no? We still need it while rendering the README?

Reply:
right, I didn't notice it was the gha for readme rendering

Reply:
No worries.

It's strange that that workflow doesn't run on PRs when it is supposed to:

https://github.com/easystats/easystats/blob/7ea13d13b6ed6f5e4b6847c17ef8b09b2f9be412/.github/workflows/render-readme.yaml#L8-L10

Reply:
I think it's because of the `paths` argument. `README.Rmd` wasn't changed so the action doesn't trigger

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore=

Reply:
Can you try removing the `paths` entry? Just to check if that triggers the workflow? 

Ideally, the action should run for every PR, doesn't matter whether README was changed or not.

Reply:
that was it, the action for the readme runs now

Reply:
So awesome! Thanks.